crates/rattler/src/validation.rs

```diff
@@ -33,6 +33,9 @@ pub enum PackageValidationError {
 
     #[error("the path '{0}' seems to be corrupted")]
     CorruptedEntry(PathBuf, #[source] PackageEntryValidationError),
+
+    #[error("failed to verify index.json")]
+    InvalidIndexJson(#[source] std::io::Error),
```

Comment:
I am wondering if we should call this `ReadIndexJsonError` (like `ReadPathsJsonError`) – or rename that one to `InvalidPathsJson`?

Reply:
Yeah good one, I renamed it to `ReadIndexJsonError` and changed the error message as well.